media-libs/mesa/metadata.xml

```diff
@@ -11,7 +11,9 @@
     <flag name="gles2">Enable GLESv2 support.</flag>
     <flag name="llvm">Enable LLVM backend for Gallium3D.</flag>
     <flag name="lm-sensors">Enable Gallium HUD lm-sensors support.</flag>
-    <flag name="opencl">Enable the Clover Gallium OpenCL state tracker.</flag>
+    <flag name="opencl">Enable the Rusticl or Clover Gallium OpenCL state tracker.</flag>
+    <flag name="rusticl">Enable the Rusticl OpenCL state tracker.</flag>
+    <flag name="clover">Enable the Clover Gallium OpenCL state tracker.</flag>
```

Comment:
Can we drop `clover` entirely? I'm okay with that. I don't think it can be used for really anything. That would let us get rid of both `IUSE=rusticl` as well, and just have rusticl controlled by `IUSE=opencl`.

Reply:
I'm in favor of dropping clover, it will make the ebuild simpler. The only downside is that R600 users will loose OpenCL for a while (rusticl is planned for R600 but still not working). I guess they can use mesa 23.0 until that happens.

Reply:
Yeah, I think that's fine. I expect the number of R600 Gentoo users using OpenCL 1.1 is approaching zero :)